### ISSUE TITLE
[EXCEPTION] Tool throw exception while dumping 

### ISSUE THREAD

**Issue by puniaze** [dirkjanm/ldapdomaindump#39]
Hi,

I have run tool with expected arguments but after some time it throws an exception.

I guess it is due to unexpected output comes from DC as a result of used LDAP queries.

This is what I get:

```
[*] Connecting to host...
[*] Binding to host
[+] Bind OK
[*] Starting domain dump
Traceback (most recent call last):
  File "/home/abdulla/dassaq/bin/ldapdomaindump", line 3, in <module>
    ldapdomaindump.main()
  File "/home/abdulla/dassaq/lib/python3.6/site-packages/ldapdomaindump/__init__.py", line 944, in main
    dd.domainDump()
  File "/home/abdulla/dassaq/lib/python3.6/site-packages/ldapdomaindump/__init__.py", line 416, in domainDump
    self.groups = self.getAllGroups()
  File "/home/abdulla/dassaq/lib/python3.6/site-packages/ldapdomaindump/__init__.py", line 253, in getAllGroups
    self.connection.extend.standard.paged_search(self.root, '(objectClass=group)', attributes=ldap3.ALL_ATTRIBUTES, paged_size=500, generator=False)
  File "/home/abdulla/dassaq/lib/python3.6/site-packages/ldap3/extend/__init__.py", line 125, in paged_search
    paged_criticality)
  File "/home/abdulla/dassaq/lib/python3.6/site-packages/ldap3/extend/standard/PagedSearch.py", line 142, in paged_search_accumulator
    paged_criticality):
  File "/home/abdulla/dassaq/lib/python3.6/site-packages/ldap3/extend/standard/PagedSearch.py", line 68, in paged_search_generator
    None if cookie is True else cookie)
  File "/home/abdulla/dassaq/lib/python3.6/site-packages/ldap3/core/connection.py", line 839, in search
    response = self.post_send_search(self.send('searchRequest', request, controls))
  File "/home/abdulla/dassaq/lib/python3.6/site-packages/ldap3/strategy/sync.py", line 139, in post_send_search
    responses, result = self.get_response(message_id)
  File "/home/abdulla/dassaq/lib/python3.6/site-packages/ldap3/strategy/base.py", line 411, in get_response
    if self.do_search_on_auto_range(self._outstanding[message_id], response):
  File "/home/abdulla/dassaq/lib/python3.6/site-packages/ldap3/strategy/base.py", line 735, in do_search_on_auto_range
    self.do_next_range_search(request, resp, attr_name)
  File "/home/abdulla/dassaq/lib/python3.6/site-packages/ldap3/strategy/base.py", line 688, in do_next_range_search
    response['raw_attributes'][attr_type] += current_response['raw_attributes'][attr_name]
TypeError: list indices must be integers or slices, not str
```

![image](https://user-images.githubusercontent.com/7716515/104496422-a8078600-55e1-11eb-9df7-a140371386a5.png)

**Comment by dirkjanm** [comment]
This seems to be related to this: https://github.com/cannatag/ldap3/issues/854
And is potentially fixed by updating ldap3 to 2.8.1

Let me know if that solves it

**Comment by dirkjanm** [comment]
closing due to lack of feedback

**Comment by mirchr** [comment]
@dirkjanm I know this is an old issue but I hit the exact same stack and resolved it by updating the `ldap3` module from v2.8 to v2.8.1.
